src/main/java/com/github/maxopoly/finale/listeners/PearlCoolDownListener.java

```diff
@@ -116,9 +129,10 @@ public void run() {
 		}
 	}
 
+	private DecimalFormat df = new DecimalFormat("#.#");
```

Comment:
I used 0.0 instead of #.#, because 0 guarantees a digit, which display 9 as 9.0, which is good because it keeps the score board text from flickering as it moves, because it gets shorter.